app/assets/javascripts/validation.coffee

```diff
@@ -176,24 +176,21 @@ $(document).ready ->
       when 'minimum' then "#{field} is too low"
       when 'parameter-range-later' then "#{field} must be later than Parameter Range Begin"
       when 'parameter-range-larger' then "#{field} must be larger than Parameter Range Begin"
-      when 'oneOf' # TODO check and remove 'Party' - was only for organization or personnel
-        # oneOf Party means it wants oneOf OrganizationName or Person
-        # Those errors don't matter to a user because they don't see
-        # that difference in the forms
-        if field == 'Party'
-          "Party is incomplete"
-        else
-          "#{field} should have one type completed"
+      when 'oneOf'
+        # the stock message is: "should match exactly one schema in oneOf", which is not as useful to the user
+        "#{field} should have one option completed"
       when 'invalidPicklist' then "#{field} #{error.message}"
       when 'enum' then "#{field} value [#{$('#' + error.id + ' option:selected').text()}] does not match a valid selection option"
-      when 'anyOf' then "#{error.message}"
+      when 'anyOf'
+        # the stock message is: "should match some schema in anyOf", which is not as useful to the user
+        "#{field} should have one schema option completed"
```

Comment:
will the user know what a schema is?

Reply:
This is the message the PO approved. We have a ticket next sprint to improve the validation messages presented to the user.